web/server/codechecker_server/server.py

```diff
@@ -609,6 +610,30 @@ def connect(self, init_db=False):
             self.__db_status = DBStatus.FAILED_TO_CONNECT
             self.__last_connect_attempt = (datetime.datetime.now(), str(ex))
 
+    def get_details(self):
+        """
+        Get details from the product.
```

Comment:
```suggestion
        Get details for a product from the database.
```

It might be worth to mention that SQLAlchemy exceptions might happen in case of some database error.

Reply:
I leave a code comment where we call this function.